catboost/python-package/ut/medium/run_catboost.py

```diff
@@ -12,6 +12,7 @@ def test():
     model = CatBoostClassifier(iterations=5)
     model.fit(X, y, cat_features=categorical_features_indices)
     preds = model.predict(X)
+    pred_single = model.predict(X[0])
```

Comment:
Please add two new tests instead of modifying the existing tests

1. cat features, e.g. adult + fit() + predict(object) == predict([object])[0]
2. only float features, e.g. Higgs + fit() + predict(pool)[0] == predict(pool[0])